SofaKernel/modules/SofaHelper/src/sofa/helper/system/PluginManager.cpp

```diff
@@ -370,18 +370,32 @@ std::string PluginManager::findPlugin(const std::string& pluginName, const std::
     return std::string();
 }
 
-bool PluginManager::pluginIsLoaded(const std::string& plugin)
+bool PluginManager::pluginIsLoaded(const std::string& pluginName)
 {
-    std::string pluginPath = plugin;
+    std::string pluginPath = pluginName;
```

Comment:
I'd stick to `plugin` as parameter, `pluginIsLoaded` is sometimes called with a plugin path.

Reply:
oki